Microsoft.Identity.Web/ClaimConstants.cs

```diff
@@ -19,5 +19,8 @@ public static class ClaimConstants
         // Newer scope claim
         public const string Scp = "scp";
         public const string Roles = "roles";
+        // Newer policy claim
+        public const string Tfp = "tfp";
```

Comment:
i can make the changes in MSAL.NET so we don't need this anymore, but this should not be surfaced to the developer anymore. will it just be used in MS.Identity.Web?

Reply:
@jennyf19 so far this const is only being used in the Identity.Web project to get the claim from `ClaimsPrincipal.cs`. How would be this in MSAL side?

Reply:
@TiagoBrenck I wasn't sure if the developer would use this value anywhere. As long as it's internal, that's fine. In MSAL, we ask the developer to include it in the authority url, and we check that `/tfp/` is included to assure it's a b2c authority, but we can remove this check and requirement in MSAL.NET